tests/install-litmus_test.go

```diff
@@ -110,7 +120,7 @@ var _ = Describe("BDD of litmus installation", func() {
 				}
 			}
 
-			fmt.Println("chaos-operator created successfully")
+			fmt.Println("Chaos Operator created successfully")
```

Comment:
Operator manifest contains ENV which contains runner's image also. Can you also update that value on the basis of provided tag so that every components ci/latest can be tested ci/latest

Reply:
That is in comment
https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/operator.yaml

Reply:
For the runner pod, `latest` is the default tag so no needed for this env if tag==latest but if the tag is ci then can you uncomment this before applying?